exe/ruby-lsp

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+# We should make sure that, if we're running on a bundler project, that it has a Gemfile.lock
+if File.exist?("Gemfile") && !File.exist?("Gemfile.lock")
+  warn("You are missing your Gemfile.lock. Run `bundle install` and restart the server")
+  exit(1)
```

Comment:
Let's use `78`, which indicates a configuration error. Being specific will make it easier to handle from the VS Code extension, since we do not want to try to restart the server automatically on this scenario.
```suggestion
  exit(78)
```